code/modules/food_and_drinks/food/snacks_frozen.dm

```diff
@@ -61,144 +61,166 @@
 //SNOWCONES//
 /////////////
 
-/obj/item/reagent_containers/food/snacks/snowcones //We use this as a base for all other snowcones
-	name = "flavorless snowcone"
-	desc = "It's just shaved ice. Still fun to chew on."
+/obj/item/reagent_containers/food/snacks/snowcones //We use this as a base for all other snowcones (I feel that this is extremely outdated as it forced all snowcones to have no contents other than water)
 	icon = 'icons/obj/food/frozen_treats.dmi'
-	icon_state = "flavorless_sc"
 	trash = /obj/item/reagent_containers/food/drinks/sillycup //We dont eat paper cups
-	bonus_reagents = list(/datum/reagent/water = 10) //Base line will allways give water
-	list_reagents = list(/datum/reagent/water = 1) // We dont get food for water/juices
+	bonus_reagents = list(/datum/reagent/water = 10) //~~Base line will allways give water~~ This line seems to break everything, added "redundant" code to force and override flavorings, appologies
```

Comment:
🤨

Reply:
Jank inheritance code that isn't working and that I don't really know how to fix myself.

As I stated in the PR.
When an admin spawns the snowcone via menu, Contents are ONLY taken from list_reagents.
When crafted from the in game menu, contents are ONLY taken from bonus_reagents.

I tinkered for like 3 days and nothing worked, food inheritance just seems broken as a whole.